.github/workflows/build.yaml

```diff
@@ -70,15 +73,24 @@ jobs:
         uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust_toolchain }}
-          targets: ${{ matrix.target_triple }}
+          targets: ${{ matrix.target_triple.name }}
 
       - name: Run Cargo Build
-        run: cargo +${{ matrix.rust_toolchain }} build --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple }} --workspace
+        run: cargo +${{ matrix.rust_toolchain }} build --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple.name }} --workspace
 
+      # Steps to use cargo-make to build and package drivers
       - name: Install Cargo Make
         uses: taiki-e/install-action@v2
         with:
           tool: cargo-make
 
       - name: Build and Package Sample Drivers
-        run: cargo make default +${{ matrix.rust_toolchain }} --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple }}
+        run: cargo make default +${{ matrix.rust_toolchain }} --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple.name }}
+
+      # Steps to use cargo-wdk to build and package drivers (skipping for arm64 as the base runner image is x86_64)
+      - name: Install cargo-wdk binary
+        run: cargo +${{ matrix.rust_toolchain }} install --git https://github.com/svasista-ms/windows-drivers-rs.git cargo-wdk --branch integrate-cargo-wdk --locked --force --profile ${{matrix.cargo_profile}}
```

Comment:
Note: The cargo-wdk used in the pipeline is taken from https://github.com/svasista-ms/windows-drivers-rs and from integrate-cargo-wdk branch as https://github.com/microsoft/windows-drivers-rs/pull/306 is still not merged. This pipeline needs to get updated with cargo-wdk from https://github.com/microsoft/windows-drivers-repo once that is merged.

Reply:
The pipeline now points to cargo-wdk on the microsoft/windows-drivers-rs main line.